### DISCUSSION TITLE
How is OneDriveGUI making onedrive autostart ?

### DISCUSSION THREAD

**Discussion by barbudor** [bpozdena/OneDriveGUI#68] [General]
Hello 

I'm testing OneDriveGUI and made onedrive autostart
But I messed up some things and wanted to restart from scratch.
So I removed folders `~ /.config/onedrive-gui` and `~/.config/onedrive/accounts` 

But now onedrive is still autostart but OneDriveGUI doesn't see it.
So I have to undo also what the initial setup one OneDriveGUI has done for sutostarting onedrive but I can't find how it is done
I've search for a systemd onedrive@USER.service but there isn't
I see nothing neither in /etc/init.d

Could you please let me know how to fully clean-up in order to restart from scratch ?

Thanks

**Comment by barbudor** [comment]
Answering to myself : it's running as a user service `systemctl --user xxxx onedrive.service`

**Reply by bpozdena** [reply]
Yes, just like it says in the [OneDrive Client documentation ](https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#onedrive-service-running-as-root-user-via-systemd-arch-ubuntu-debian-opensuse-fedora) :-) .

**Reply by barbudor** [reply]
Missed it 🙄
Thanks
